### PR TITLE
Fix postman workflow

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
after initial merge of postman workflow onto main, can test and fix issues.

**Checklist**
- [ ] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Comment by jeffreyscarpenter** [comment]
Note the workflow will still show as failing but that is due to actual failures in the postman script, as @johnsmartco is adding tests to it.

File: .github/workflows/performance-testing.yaml [hunk 1]
```diff
@@ -74,11 +81,13 @@ jobs:
       - name: Start Docker Compose
         run: |
           JSONTAG=$(git show -s --format="%H")
-          docker pull ${{ secrets.ECR_REPOSITORY }}/stargateio/${{ matrix.docker-image }}:$JSONTAG
-          docker image tag ${{ secrets.ECR_REPOSITORY }}/stargateio/${{ matrix.docker-image }}:$JSONTAG stargateio/${{ matrix.docker-image }}:$JSONTAG
+          docker pull ${{ secrets.ECR_REPOSITORY }}/stargateio/jsonapi:$JSONTAG
```
Comment (jeffreyscarpenter) [inline .github/workflows/performance-testing.yaml:84]: In copying steps from docker pull from this workflow for the new postman workflow, I realized that this workflow wasn't actually pulling the coordinator image from ECR. I updated both workflows with logic to pull the coordinator image according to the Stargate version referenced in the pom.

File: .github/workflows/performance-testing.yaml [hunk 2]
```diff
@@ -76,9 +83,11 @@ jobs:
           JSONTAG=$(git show -s --format="%H")
           docker pull ${{ secrets.ECR_REPOSITORY }}/stargateio/${{ matrix.docker-image }}:$JSONTAG
           docker image tag ${{ secrets.ECR_REPOSITORY }}/stargateio/${{ matrix.docker-image }}:$JSONTAG stargateio/${{ matrix.docker-image }}:$JSONTAG
+          SGTAG="$(./mvnw -f . help:evaluate -Dexpression=stargate.int-test.coordinator.image-tag -q -DforceStdout)"
```
Comment (jeffreyscarpenter) [inline .github/workflows/performance-testing.yaml:86]: In copying steps from docker pull from this performance testing workflow for the new postman workflow, I realized that this workflow wasn't actually pulling the coordinator image from ECR. I updated both workflows with logic to pull the coordinator image according to the Stargate version referenced in the pom.

File: .github/workflows/performance-testing.yaml [hunk 3]
```diff
@@ -55,6 +55,13 @@ jobs:
         with:
           ref: main
 
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          cache: maven
+
```
Comment (ivansenic) [inline .github/workflows/performance-testing.yaml:64]: seems this is added, but not required by any change below?

Comment (ivansenic) [reply .github/workflows/performance-testing.yaml:64]: ah is it for the `/mvnw`?

**Review by ivansenic — APPROVED** [review]

**Pull request merged**